.eslintrc

```diff
@@ -37,3 +37,4 @@ rules:
   no-unused-expressions: [0]
   no-use-before-define: [2, nofunc]
   yoda: [0]
+  linebreak-style: [0]
```

Comment:
Git is probably converting the endings automatically. We can leave this line in for now and fix it later with a `.gitattributes` file.

For reference see https://help.github.com/articles/dealing-with-line-endings/